core/src/sigverify_stage.rs

```diff
@@ -24,7 +24,9 @@ use {
     thiserror::Error,
 };
 
-const MAX_SIGVERIFY_BATCH: usize = 10_000;
+/// Maximum number of valid, filtered packets that can be sent along for
+/// signature verification.
+const MAX_SIGVERIFY_VALID_PACKETS: usize = 10_000;
```

Comment:
Naming: My first intuition is that a packet is valid only after sigverify is done with it. This is about the max number of packets we'll run verify on. Bad suggestions: `MAX_SIGVERIFY_VERIFY_PACKETS`, `MAX_SIGVERIFY_PACKETS_TO_VALIDATE`, `MAX_PACKETS_TO_VALIDATE`, `MAX_SIGVERIFY_INPUT_PACKETS`.

It's confusing because sigverify is both sigverify-stage, which includes dedup, and the signature verification part of the sigverify-stage.

Reply:
Do you think this is clear?
```rust
/// Maximum number of filtered packets that can have signatures verified per loop.
const MAX_VERIFIED_PACKETS_PER_LOOP: usize = 10_000;
```